src/python/pants/notes/master.rst

```diff
@@ -1,9 +1,40 @@
 Master Pre-Releases
 ===================
 
+1.10.0.dev4 (08/24/2018)
```

Comment:
Is this an intentional precedent change? Previously the "This document describes ``dev``..." sentence preceeded all releases.

Reply:
Ah, copy-pasta error. Good spot.